docs/core-concepts/api-calls-anatomy/index.md

```diff
@@ -47,62 +57,113 @@ Every request data has several required fields which you must provide them and t
+                        "documents_supported": {},
+                        "has_visual_sample": 0,
+                        "is_country_supported": 0
+                    },
+                    "onfido": {
+                        "documents_supported": {
+                            "driving_licence": {
+                                "display_name": "Driving Licence"
+                            }
+                        },
+                        "is_country_supported": 0
+                    }
+                }
+            },
+            "phone_idd": "35818",
+            "text": "Aland Islands",
+            "value": "ax"
+        },
+        {
+            "identity": {
+                "services": {
+                    "idv": {
+                        "documents_supported": {},
+                        "has_visual_sample": 0,
+                        "is_country_supported": 0
+                    },
+                    "onfido": {
+                        "documents_supported": {
+                            "driving_licence": {
+                                "display_name": "Driving Licence"
+                            },
+                            "national_identity_card": {
+                                "display_name": "National Identity Card"
+                            },
+                            "passport": {
+                                "display_name": "Passport"
+                            }
+                        },
+                        "is_country_supported": 1
+                    }
+                }
+            },
+            "phone_idd": "355",
+            "text": "Albania",
+            "tin_format": [
+                "^[A-Ta-t0-9]\\d{8}[A-Wa-w]$"
+            ],
+            "value": "al"
+        },
+        // ....
+  ],
 };
 ```
 
-Here the `residence_list` is the `Identifier Field` and it contains the actual data you requested. the array is removed here for brevity sake, you can check the actual real response [here](https://api.deriv.com/api-explorer#residence_list).
+Here the `residence_list` is the `method name` and it contains the actual data you requested. rest of the array is removed here for brevity sake, you can check the actual response [here](https://api.deriv.com/api-explorer#residence_list).
 
-### The `echo_req` Field
+#### The `echo_req` Field
 
-As you can see this `Field` contains the exact `Request Data` you sent to the server.
+This `Field` contains the exact `Request Data` you sent to the server.
 
-### The `msg_type` Field
+#### The `msg_type` Field
 
-This `Field` help you understand what kind of message you're getting on `message` event of the websocket connection. for example your `onmessage` event handler can be something like this:
+This `Field` helps you determine which message data you're getting on `message` event of the websocket connection. for example your `onmessage` event handler for you websocket connection is `Javascript` would be:
```

Comment:
Fixing spelling and typos:

This `Field` helps you determine which message data you're getting on the `message` event of the websocket connection. For example, your `onmessage` event handler for your websocket connection in `Javascript` would be: